### ISSUE TITLE
IUIA5: NVDA 2017.4 does not come with IUIAutomation5 interface in UIA handler

### ISSUE THREAD

**Issue by josephsl** [josephsl/wintenApps#42]
Hi,

Received multiple reports regarding this: if recent add-on releases are run from NVDA 2017.4, it won't work because of missing IUIA5.

As this is a critical issue, an immediate fix will be deployed. Thanks.

**Comment by josephsl** [comment]
Fixed as part of 18.03.2.
